src/assets/pages/blog.json

```
@@ -1,7 +1,16 @@
+      "banner": "https://ok.org.br/wp-content/uploads/2025/09/banners_catarse_qd_13-1.png",
+      "date": "09/17/2025",
+      "author": "Open Knowledge Brasil"
+    }
+    {
```

Falta uma vírgula para separar os elementos da lista

```suggestion
    },
    {
```